diff -Naur xdg-autostart-0.1/libxdgutils/XDGMenu.cpp xdg-autostart-0.1.tpg/libxdgutils/XDGMenu.cpp
--- xdg-autostart-0.1/libxdgutils/XDGMenu.cpp	2011-02-12 19:54:56.000000000 +0000
+++ xdg-autostart-0.1.tpg/libxdgutils/XDGMenu.cpp	2016-02-03 18:43:49.736311668 +0000
@@ -187,8 +187,12 @@
     "GNOME",
     "KDE",
     "LXDE",
+    "LXQt",
     "ROX",
     "XFCE",
+    "MATE",
+    "UNITY",
+    "OPENBOX",
     "Old",
     NULL
 };
