diff -urN xdg-autostart-0.1/autostart/xdg-autostart.cpp xdg-autostart-0.1-patched/autostart/xdg-autostart.cpp
--- xdg-autostart-0.1/autostart/xdg-autostart.cpp	2011-02-11 19:45:49.000000000 +1000
+++ xdg-autostart-0.1-patched/autostart/xdg-autostart.cpp	2012-09-01 16:08:33.756674878 +1100
@@ -22,6 +22,7 @@
  */
 
 #include <iostream>
+#include <unistd.h>
 
 #include "XDGAutostart.h"
 #include "XDGUtils.h"
diff -urN xdg-autostart-0.1/test/manual.cpp xdg-autostart-0.1-patched/test/manual.cpp
--- xdg-autostart-0.1/test/manual.cpp	2011-02-11 19:45:49.000000000 +1000
+++ xdg-autostart-0.1-patched/test/manual.cpp	2012-09-01 15:59:22.481680719 +1100
@@ -22,6 +22,7 @@
  */
 
 #include <iostream>
+#include <unistd.h>
 
 #include "XDGAutostart.h"
 #include "XDGBasedir.h"
